avocado/plugins/resolvers.py

```diff
@@ -125,7 +126,9 @@ def resolve(reference):
         if criteria_check is not True:
             return criteria_check
 
+        runnable = Runnable('tap', reference,
+                            config=settings.filter_config(self.config,
+                                                          r'^runner\.'))
```

Comment:
After #4688, we are passing the entire config to the runners. So what you think about using the same approach here?

Reply:
I was thinking about it, but I wanted to keep the same functionality as it is and just fix the bug. I see this as a different problem, so I don't wanted to  put it in the same PR, but if you this that is ok I can also delete the filtering.